src/components/range-choice/style.scss

```diff
@@ -31,62 +31,156 @@ $border-radius: 8px;
   margin: 0 $thumb-width/2;
 }
 
-.answer-text {
-  display: block;
-  position: absolute;
-  background: #eee;
-  color: #000;
-  padding: 3px 5px;
-  border-radius: 3px;
-  font-size: 0.8rem;
-  transform: translate(-50%, 0);
-  top: 22px;
-  z-index: 2;
-  white-space: nowrap;
-  text-align: center;
-
-  &::after {
-    content: '';
-    position: absolute;
-    left: 50%;
-    top: 100%;
-    height: 9px;
-    border-left: 1px dotted #ccc;
+.tick {
+  width: 20px;
+  height: auto;
+  path + path {
+    fill: $answer-correct-label-background-color;
   }
 }
 
-.answer-correct {
-  transition: left 0.5s, opacity 0.5s;
-  top: 0;
-  pointer-events: none;
-  background: #ccc;
-  z-index: 1;
-  &::after {
-    height: 39px;
+// Reference: https://codepen.io/thebabydino/pen/WdeYMd
+
+// CONTROL
+$track-w: 100%;
+$track-h: 16px;
+$thumb-w: 20px;
+$thumb-h: 34px;
+$track-c: $answer-background-color;
+
+@mixin track() {
+  box-sizing: border-box;
+  border: none;
+  width: $track-w;
+  height: $track-h;
+  background: $track-c;
+}
+
+@mixin thumb() {
+  box-sizing: border-box;
+  border: none;
+  width: $thumb-w;
+  height: $thumb-h;
+  border-radius: 0;
+  background: $answer-label-background-color;
+  box-shadow: 0 0 0 #000, 0 0 0 #0d0d0d;
+  border: 2px solid #fff;
+}
+
+.wrap {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: $track-w;
+  height: 3.5 * $thumb-h;
+  font: 1em/1 arial, sans-serif;
+}
+
+.wrap [type="range"] {
+  &,
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+  }
+
+  flex: 1;
+  margin: 0;
+  padding: 0;
+  min-height: $thumb-h;
+  background: transparent;
+  font: inherit;
+
+  &::-webkit-slider-runnable-track {
+    @include track();
   }
-  .tick {
+  &::-moz-range-track {
+    @include track;
+  }
+  &::-ms-track {
+    @include track;
+  }
+
+  &::-webkit-slider-thumb {
+    margin-top: 0.5 * ($track-h - $thumb-h);
+    @include thumb;
+  }
+  &::-moz-range-thumb {
+    @include thumb;
+  }
+  &::-ms-thumb {
+    margin-top: 0;
+    @include thumb;
+  }
+
+  &::-ms-tooltip {
+    display: none;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  ~ output {
+    display: block;
```

Comment:
IE doesn't know anything about the `<output>` tag, but it looks like you've got enough styling for it to not be an issue ther